[2023-09-22T16:52:34.718+0000] {logging_mixin.py:149} WARNING - /home/***/.local/lib/python3.9/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-09-22T16:52:35.029+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_taskflow_api_v02.get_name scheduled__2021-11-21T00:00:00+00:00 [queued]>
[2023-09-22T16:52:35.054+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_taskflow_api_v02.get_name scheduled__2021-11-21T00:00:00+00:00 [queued]>
[2023-09-22T16:52:35.057+0000] {taskinstance.py:1331} INFO - Starting attempt 1 of 6
[2023-09-22T16:52:35.094+0000] {taskinstance.py:1350} INFO - Executing <Task(_PythonDecoratedOperator): get_name> on 2021-11-21 00:00:00+00:00
[2023-09-22T16:52:35.117+0000] {standard_task_runner.py:57} INFO - Started process 592 to run task
[2023-09-22T16:52:35.150+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'dag_with_taskflow_api_v02', 'get_name', 'scheduled__2021-11-21T00:00:00+00:00', '--job-id', '146', '--raw', '--subdir', 'DAGS_FOLDER/test2.py', '--cfg-path', '/tmp/tmp811w03wt']
[2023-09-22T16:52:35.168+0000] {standard_task_runner.py:85} INFO - Job 146: Subtask get_name
[2023-09-22T16:52:35.233+0000] {logging_mixin.py:149} WARNING - /home/***/.local/lib/python3.9/site-packages/***/settings.py:188 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-09-22T16:52:35.363+0000] {task_command.py:410} INFO - Running <TaskInstance: dag_with_taskflow_api_v02.get_name scheduled__2021-11-21T00:00:00+00:00 [running]> on host 75cee4990e54
[2023-09-22T16:52:35.411+0000] {logging_mixin.py:149} WARNING - /home/***/.local/lib/python3.9/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-09-22T16:52:37.311+0000] {taskinstance.py:1568} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='coder2j' AIRFLOW_CTX_DAG_ID='dag_with_taskflow_api_v02' AIRFLOW_CTX_TASK_ID='get_name' AIRFLOW_CTX_EXECUTION_DATE='2021-11-21T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-11-21T00:00:00+00:00'
[2023-09-22T16:52:37.314+0000] {python.py:183} INFO - Done. Returned value was: {'first_name': 'Jerry', 'last_name': 'Fridman'}
[2023-09-22T16:52:37.450+0000] {taskinstance.py:1368} INFO - Marking task as SUCCESS. dag_id=dag_with_taskflow_api_v02, task_id=get_name, execution_date=20211121T000000, start_date=20230922T165235, end_date=20230922T165237
[2023-09-22T16:52:37.565+0000] {local_task_job_runner.py:232} INFO - Task exited with return code 0
[2023-09-22T16:52:38.293+0000] {taskinstance.py:2674} INFO - 0 downstream tasks scheduled from follow-on schedule check
